frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -108,6 +122,16 @@ function EmbedFrame({
 
   const hasHeader = Boolean(finalName || hasParameters);
 
+  const applyFilterButton = shouldRenderApplyButton ? (
```

Comment:
Fix passing `ReactNode` by constructing the button inside the view components instead.

I try to also make it as clear as possible which props are for auto-apply filters.